scipy/differentiate/tests/test_differentiate.py

```diff
@@ -446,7 +446,38 @@ def test_saddle_gh18811(self, case):
         assert_allclose(res.df, 0, atol=atol)
 
 
-class TestJacobian:
+class JacobianHessianTest:
+    def test_iv(self):
+        jh_func = self.jh_func.__func__
+
+        # Test input validation
+        message = "Argument `x` must be at least 1-D."
+        with pytest.raises(ValueError, match=message):
+            jh_func(np.sin, 1, tolerances=dict(atol=-1))
+
+        # Confirm that other parameters are being passed to `_derivative`,
```

Comment:
Not sure what  `` `_derivative` `` refers to?

Reply:
It used to be private. 
```suggestion
        # Confirm that other parameters are being passed to `derivative`,
```

Reply:
`derivative` is also nice in that it is clearer (to me at least) it is elementwise and that it is not the gradient

Reply:
OK, I'll open a separate PR after this one.

Reply:
Julia also favors `derivative`
https://docs.sciml.ai/FiniteDiff/stable/#Scalar-Derivatives
https://juliadiff.org/ForwardDiff.jl/stable/user/api/#Derivatives-of-f(x::Real)::Union{Real,AbstractArray}